kotlintest-assertions/src/main/kotlin/io/kotlintest/matchers/collections/matchers.kt

```diff
@@ -62,11 +51,6 @@ fun <T> List<T>.shouldHaveElementAt(index: Int, element: T) = this should haveEl
 
 fun <T> List<T>.shouldNotHaveElementAt(index: Int, element: T) = this shouldNot haveElementAt(index, element)
 
-@Deprecated("Use shouldHaveElementAt", ReplaceWith("this.shouldHaveElementAt(index, element)", "io.kotlintest.should"))
-fun <T, L : List<T>> L.shouldContainElementAt(index: Int, element: T) = this should haveElementAt(index, element)
```

Comment:
Are these relatively recent deprecations ?

Reply:
05/2018

Reply:
So all these deprecations went into 3.1.0 so I think we can remove
I just don't want to remove anything we only deprecated in 3.2.0 as we only released that a month ago.
And we released 3.1.0 on 11th May - so anything deprecated after that might not have been seen by a user until Jan this year.